app/themes/user-admin/e2e/user-create/user-create.e2e-spec.ts

```diff
@@ -0,0 +1,77 @@
+import { UserAdminPage } from './user-create.po';
+import { protractor, browser, element, by, WebElement } from 'protractor';
+
+describe('User Create', function() {
+    let page: UserAdminPage;
+
+    beforeEach(() => {
+        page = new UserAdminPage();
+    });
+
+    it('should login successfully with right email and password', () => {
+        page.navigateTo("/");
+        let email = browser.findElement(protractor.By.name('email'));
+        let password = browser.findElement(protractor.By.name('password'));
+        let submit = browser.findElement(protractor.By.className('btn btn-success'));
+
+        email.sendKeys('foo@mail.com');
+        password.sendKeys('asdf1234');
+        
+        submit.click();
+
+        expect(page.getParagraphText("app-home h1")).toEqual('Welcome to the Erdiko User Admin');
+    });
+
+    it('should lead to Create a User page at click of the Create a User link', () => {
+        //check for both User List Link
+        let create = browser.findElement(protractor.by.css('app-home ul > li:first-child > a'));
+        create.click();
+    });
+
+    it('should lead to User List page at click of the cancel button', () => {
+        let cancel = browser.findElement(protractor.By.css('button[type=cancel]'));
+        cancel.click();
+
+        //Go back to Create User page by clicking 'Create a New User'
+        let create = browser.findElement(protractor.By.cssContainingText('button', 'Create a New User'));
+        create.click();
+    });
+
+    it('should input name, email, role and save the user and check its presence', () => {
+        let name = browser.findElement(protractor.By.name('name'));
+        let email = browser.findElement(protractor.By.name('email'));
+        let user = browser.findElement(protractor.By.cssContainingText('option', 'User'));
+        let admin = browser.findElement(protractor.By.cssContainingText('option', 'Admin'));
+        let save = browser.findElement(protractor.By.cssContainingText('button', 'Save'));
+
+        let cancel = browser.findElement(protractor.By.cssContainingText('button', 'Cancel'));
+
+        //Create new user
+        name.sendKeys('Elliot Alderson');
+        email.sendKeys('elliot@funsociety.com');
+        admin.click();
+        save.click();
+
+        cancel.click();
+
+        //NOTE. clicking cancel with following config will cause save button to be clicked twice.
```

Comment:
We might need to add a `e.stopPropagation()` to the save button function. Try that out and see if it works, make sure the unit tests continue to run as expected.